api/oh.yaml

```diff
@@ -94,7 +94,7 @@ tags:
   - name: ward-controller
     description: Ward Controller
 schemes:
-  - https
+  - http
```

Comment:
is this change meant to be? for me would very good but... you find the way to customize this parameter for different env (develop, test, production, etc...)?